### DISCUSSION TITLE
ip access prevent login

### DISCUSSION THREAD

**Discussion by Enhed** [directus/directus#10080] [Feature Requests]
If set ip access in role, users can login without equals ip

**Comment by benhaynes** [comment]
I saw another issue from today that was saying this didn't work through Load Balancers... not sure if related. Either way, if this is a bug it should be reported as an issue, not a discussion. But the issue would need more info to accurately reproduce. 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
